pkg/controller/volume/attachdetach/cache/actual_state_of_world_test.go

```diff
@@ -26,6 +26,36 @@ import (
 	volumetesting "k8s.io/kubernetes/pkg/volume/testing"
 )
 
+func verifyAttachedVolume(
```

Comment:
Got it. Let's keep Tests grouped together at the top, and helper methods at the bottom.